<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chaxun</class>
 <widget class="QWidget" name="chaxun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>717</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>推荐专业查询</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>430</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>该地区排名：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>280</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪蝶语体简</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 255, 246);
font: 15pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>东校区</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>南校区</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>北校区</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>珠海校区</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>深圳校区</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>430</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>210</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>地区选择：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>510</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>文理科选择：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>280</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;校区选择：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>430</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>第</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>210</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪蝶语体简</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 255, 246);
font: 15pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>北京</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>上海</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>广东</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>河北</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>山东</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>辽宁</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黑龙江</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>吉林</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>青海</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>河南</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>江苏</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>湖北</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>湖南</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>浙江</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>江西</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>云南</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>福建</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>台湾</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>海南</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>山西</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>四川</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>陕西</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>贵州</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>安徽</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>重庆</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>天津</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>内蒙古</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>广西</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>西藏</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>宁夏</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新疆</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>澳门</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>香港</string>
    </property>
   </item>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>430</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪蝶语体简</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 255, 246);
font: 18pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>510</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪蝶语体简</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 255, 246);
font: 15pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>理科</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>文科</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>600</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/heidi.png);
color: rgb(255, 255, 255);
font: 15pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_4">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>350</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪蝶语体简</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 255, 246);

font: 15pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>计算机</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>材料</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>政治</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>医学</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>环境</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>哲学</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>数学</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>物理</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>化学</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>金融</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>法学</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>材料</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>历史</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>350</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>爱好方向：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>20</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/timg.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>100</y>
     <width>1141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="midLineWidth">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="LinkButton_news_zhuye">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>187</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪蝶语体简</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 255, 246);
font: 15pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
   <property name="text">
    <string>主页</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="LinkButton_news_zhuye_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪蝶语体简</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 255, 246);
font: 15pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
   <property name="text">
    <string>专业信息查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>501</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/1.jpg);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image:url(:/image/timg.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>411</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/zhuanyechaxun.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>600</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 255, 246);
font: 15pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>721</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/cool-background.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_12</zorder>
  <zorder>label_3</zorder>
  <zorder>comboBox</zorder>
  <zorder>label_5</zorder>
  <zorder>label_2</zorder>
  <zorder>label_6</zorder>
  <zorder>label</zorder>
  <zorder>label_4</zorder>
  <zorder>comboBox_2</zorder>
  <zorder>plainTextEdit</zorder>
  <zorder>comboBox_3</zorder>
  <zorder>pushButton</zorder>
  <zorder>comboBox_4</zorder>
  <zorder>label_7</zorder>
  <zorder>label_9</zorder>
  <zorder>line</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_8</zorder>
  <zorder>textBrowser</zorder>
  <zorder>LinkButton_news_zhuye_2</zorder>
  <zorder>LinkButton_news_zhuye</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>LinkButton_news_zhuye</sender>
   <signal>clicked()</signal>
   <receiver>chaxun</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
